### PR TITLE
Newarrival topic

### PR DESCRIPTION
**Pull request by akhilhbc**
@josehbc how do we do unit tests ?

### CODE REVIEW

File: local/run_app.sh [hunk 1]
```diff
@@ -23,6 +23,7 @@ export BOOTSTRAP_SERVERS="localhost:9092"
 export SCHEMA_REGISTRY="http://localhost:8081"
 export SOURCE_TOPIC="product_source"
 export SINK_TOPIC="product_sink"
+export NEWARRIVAL_TOPIC="newarrival"
```
Comment (josehbc) [inline local/run_app.sh]: looks good. Call it new_arrivals to keep the case consistent with the other values

**Comment by akhilhbc** [comment]
Merge this in now, unit tests have a separate jira

**Comment by akhilhbc** [comment]
will create a new PR when I get the updated schema

File: local/run_app.sh [hunk 2]
```diff
@@ -23,6 +23,7 @@ export BOOTSTRAP_SERVERS="localhost:9092"
 export SCHEMA_REGISTRY="http://localhost:8081"
 export SOURCE_TOPIC="product_source"
 export SINK_TOPIC="product_sink"
+export NEWARRIVAL_TOPIC="new_arrival"
```
Comment (josehbc) [inline local/run_app.sh]: NEW_ARRIVAL_TOPIC same as before just for consistency.

File: modules/product-processor/src/main/scala/com/hbc/streams/product/transformer/NewArrivalTransformerInstance.scala [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+private[transformer] trait NewArrivalTransformerInstance {
+
+  type ProductTransformer = TransformerSupplier[
+    ProductId,
+    NewArrivalProduct,
+    KeyValue[ProductId, NewArrivalProduct]
+  ]
+
+  implicit val productTransformer: ProductTransformer =
```
Comment (josehbc) [inline modules/product-processor/src/main/scala/com/hbc/streams/product/transformer/NewArrivalTransformerInstance.scala]: We don't need a new productTransformer. Instead update the existing one and add isNew:

            value.copy(isSale = p.isSale, isNew = p.isNew)

File: modules/schemas/src/main/avro/NewArrival.avsc [hunk 4]
```diff
@@ -0,0 +1,32 @@
+{
```
Comment (josehbc) [inline modules/schemas/src/main/avro/NewArrival.avsc]: We don't need a new Avro schema. The existing ProductId, Product1 can be reused.

**Pull request merged**